### PR TITLE
add distribute tutorials

### PR DESCRIPTION
**Pull request by Songyuanwei**
Thank you for your contribution to the MindOCR repo.
Before submitting this PR, please make sure:

- [x] You have read the [Contributing Guidelines on pull requests](https://github.com/mindspore-lab/mindcv/blob/main/CONTRIBUTING.md)
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved terminology
- [ ] You have added unit tests

## Motivation

(Write your motivation for proposed changes here.)

## Test Plan

(How should this PR be tested? Do you require special setup to run the test or repro the fixed bug?)

## Related Issues and PRs

(Is this PR part of a group of changes? Link the other relevant PRs and Issues here. Use https://help.github.com/en/articles/closing-issues-using-keywords for help on GitHub syntax)

### CODE REVIEW

File: docs/cn/tutorials/distribute_train_CN.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+```shell
+# n is the number of GPUs/NPUs
+mpirun --allow-run-as-root -n 2 python tools/train.py --config configs/det/dbnet/db_r50_icdar15.yaml
+```
+## 配置RANK_TABLE_FILE进行训练
+
+使用此种方法在进行分布式训练前需要创建json格式的HCCL配置文件，即生成RANK_TABLE_FILE文件，以下为生成8卡相应配置文件命令，更具体信息及相应脚本参见[hccl_tools](https://gitee.com/mindspore/models/tree/master/utils/hccl_tools)中的说明，
+``` shell
+python python hccl_tools.py --device_num "[0,8)"
```
Comment (SamitHuang) [inline docs/cn/tutorials/distribute_train_CN.md:18]: python python 重复

File: docs/cn/tutorials/distribute_train_CN.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+```
+输出为：
+```
+hccl_8p_10234567_127.0.0.1.json
```
Comment (SamitHuang) [inline docs/cn/tutorials/distribute_train_CN.md:22]: 这里给出一下 hccl_8p_10234567_127.0.0.1.json 的内容示例。

File: docs/cn/tutorials/distribute_train_CN.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+export RANK_SIZE=8
+export RANK_TABLE_FILE="./hccl_8p_01234567_127.0.0.1.json"
+
+for ((i = 0; i < ${DEVICE_NUM}; i++)); do
+    export DEVICE_ID=$i
+    export RANK_ID=$i
+    echo "Launching rank: ${RANK_ID}, device: ${DEVICE_ID}"
+    if [ $i -eq 0 ]; then
+  echo 'i am 0'
+      python -u tools/train.py --config configs/det/dbnet/db_r50_icdar15.yaml &> ./train.log &
```
Comment (SamitHuang) [inline docs/cn/tutorials/distribute_train_CN.md:42]: 说明训练其他模型要改的地方  python -u tools/train.py --config path/to/model_config.yaml

File: docs/cn/tutorials/distribute_train_CN.md (hunk 1 above)
Comment (Songyuanwei) [reply docs/cn/tutorials/distribute_train_CN.md]: fixed

File: docs/cn/tutorials/distribute_train_CN.md (hunk 2 above)
Comment (Songyuanwei) [reply docs/cn/tutorials/distribute_train_CN.md:22]: fixed

**Comment by hadipash** [comment]
Can I run in distributed mode on cards [4, 8)? Or the number must start with 0?

**Pull request merged**